core/trino-main/src/main/java/io/trino/sql/planner/OptimizerConfig.java

```diff
@@ -230,6 +231,20 @@ public OptimizerConfig setMaxReorderedJoins(int maxReorderedJoins)
         return this;
     }
 
+    @Min(1)
+    public int getMaxPrefetchedInformationSchemaPrefixes()
+    {
+        return maxPrefetchedInformationSchemaPrefixes;
+    }
+
+    @Config("optimizer.max-prefetched-information-schema-prefixes")
```

Comment:
Do you think users would be actually using that? Maybe it should not be exposed to public if you care only about test execution time.
Alternatively, should we make this config hidden?